src/ducks/GetTimeSeries/queries/get_metric.js

```diff
@@ -6,9 +6,10 @@ export const GET_METRIC = metric => gql`
     $from: DateTime!
     $to: DateTime!
     $interval: String
+    $transform: TimeseriesMetricTransformInputObject
```

Comment:
Query linked in the `[D-0-1]` comment should be modified instead. `GetTimeSeries` contains deprecated logic